modules/core/src/main/java/org/apache/ignite/internal/processors/bulkload/BulkLoadProcessor.java

```diff
@@ -120,7 +134,7 @@ public void onError(Exception failReason) {
         if (isClosed)
             return;
 
-        try {
+        try (MTC.TraceSurroundings ignored = MTC.supportContinual(qrySpan)) {
```

Comment:
Seems, that it's useful to either use separate span for batch processing (and in that case use support() instead of supportContinual() here in order to close span on bulkLoadProcessor.close()) or populate span with log points on every processBatch() and close().